pkg/scheduler/admission/podnodeconstraints/admission.go

```diff
@@ -77,6 +78,9 @@ var resourcesToCheck = map[unversioned.GroupResource]unversioned.GroupKind{
 // we choose not to handle in this plugin
 var resourcesToIgnore = []unversioned.GroupKind{
 	extensions.Kind("DaemonSet"),
+	securityapi.Kind("PodSecurityPolicySelfSubjectReview"), // TODO: should this go through admission?
```

Comment:
Yeah, let's run these.  The rejection messages will help users figure what's happening to their pods that are being created indirectly.